build.gradle

```diff
@@ -34,8 +35,23 @@ ext {
 }
 
 dependencies {
-    api 'junit:junit:4.13.2',
-            "com.vladsch.flexmark:flexmark:$flexmarkVersion",
+    api (platform("org.junit:junit-bom:$junitJupiterVersion"))
+    api 'org.junit.jupiter:junit-jupiter'
+    api 'junit:junit:4.13.2'
+    testRuntimeOnly('org.junit.vintage:junit-vintage-engine') {
+        because 'allows JUnit 3 and JUnit 4 tests to run'

```

Comment:
Nice. I hadn't come across the `because` keyword before. 

I think we should ditch the JUnit 3 runner with this update, so we just support current and current - 1 versions.